src/platforms/php/guides/laravel/other-versions/laravel5-6-7.mdx

```diff
@@ -0,0 +1,90 @@
+---
+title: Laravel 5.x, 6.x and 7.x
+description: "Learn about using Sentry with Laravel 5.x, 6.x and 7.x."
+sidebar_order: 600
+redirect_from:
+  - /platforms/php/guides/laravel/other-versions/laravel5-6/
+  - /platforms/php/guides/laravel/configuration/other-versions/laravel5-6/
+---
+
+Laravel is supported via a native package: [sentry-laravel](https://github.com/getsentry/sentry-laravel).
+
+This guide is for Laravel 5.x, 6.x and 7.x. We also provide instructions for [the latest Laravel](/platforms/php/guides/laravel/) as well as [Lumen-specific instructions](/platforms/php/guides/laravel/other-versions/lumen/).
+
+## Install
+
+Install the `sentry/sentry-laravel` package:
+
+```bash
+composer require sentry/sentry-laravel
+```
+
+If you're on Laravel 5.5 or later the package will be auto-discovered. Otherwise you will need to manually configure it in your `config/app.php`.
+
+```php {filename:config/app.php}
+'providers' => array(
+    // ...
+    Sentry\Laravel\ServiceProvider::class,
+),
+'aliases' => array(
+    // ...
+    'Sentry' => Sentry\Laravel\Facade::class,
+),
+```
+
+Configure the Sentry DSN with this command:
+
+```shell
+php artisan sentry:publish --dsn=___PUBLIC_DSN___
+```
+
+It creates the config file (`config/sentry.php`) and adds the `DSN` to your `.env` file.
+
+```shell {filename:.env}
+SENTRY_LARAVEL_DSN=___PUBLIC_DSN___
+```
+
+Add Sentry reporting to `App/Exceptions/Handler.php`.
+
+```php {filename:App/Exceptions/Handler.php}
+public function report(Exception $exception)
+{
+    if ($this->shouldReport($exception) && app()->bound('sentry')) {
+        app('sentry')->captureException($exception);
+    }
+
+    parent::report($exception);
+}
+```
+
+### Verify with Artisan
+
+You can test your configuration using the provided `sentry:test` artisan command:
+
+```shell
+php artisan sentry:test
+```
+
+
+### Verify with code
+
+You can verify that Sentry is capturing errors in your Laravel application by creating a route that will throw an exception:
+
+```php {filename:routes/web.php}
+Route::get('/debug-sentry', function () {
+    throw new Exception('My first Sentry error!');
+});
+```
+
+Visiting this route will trigger an exception that will be captured by Sentry.
+
+
+## Local development and testing
+
+When Sentry is installed in your application it will also be active when you are developing or running tests.
+
+You most likely don't want errors to be sent to Sentry when you are developing or running tests, set the DSN value to `null` to disable sending errors to Sentry.
```

Comment:
```suggestion
You most likely don't want errors to be sent to Sentry when you are developing or running tests. To avoid this, set the DSN value to `null` to disable sending errors to Sentry.
```